src/CSEjector.sol

```diff
@@ -60,12 +88,181 @@ contract CSEjector is
         _pauseFor(duration);
     }
 
+    // @inheritdoc ICSEjector
+    function processExitDelayReport(
+        uint256 nodeOperatorId,
+        bytes calldata publicKey,
+        uint256 eligibleToExitInSec
+    ) external onlyModule {
+        uint256 curveId = ACCOUNTING.getBondCurveId(nodeOperatorId);
+
+        uint256 allowedExitDelay = PARAMETERS_REGISTRY.getAllowedExitDelay(
```

Comment:
getAllowedExitDelay -> getAllowedExitDelaySec

Reply:
added note in natspec